docs/user/aws/limits.md

```diff
@@ -82,11 +82,15 @@ limit to satisfy the need. If not, please ask AWS to increase the limit via a su
 
 By default, each cluster will create 2 network load balancers for the master API server (1 internal, 1 external) and a
 single classic elastic load balancer for the router. Additional Kubernetes LoadBalancer Service objects will create
-additional [load balancers][load-balancing].
+additional [load balancers][load-balancing].  The default classic load balancer limit is
+[20 per region][classic-load-balancer-limits] and the default network load balancer limit is also
```

Comment:
WDYT about linking to the docs for default limits but not specifically calling them out?

Reply:
> WDYT about linking to the docs for default limits but not specifically calling them out?

I expect these are stable enough that I'm fine with local numbers, and the link is there for folks who want to check us.  But I'm ok dropping the local numbers if you want.